carbon-design-system/ibm-security
# feat: update to Carbon 10.10.0 release

## Affected issues

- Resolves #389 

## Proposed changes

- upgrade packages to [Carbon's 10.10.0 release](https://github.com/carbon-design-system/carbon/releases/tag/v10.10.0).
- update snapshots for updated inner Carbon core components
- update `FilterPanelSearch` tests in particular -- previously using `getByLabelText` because they only had one occurrence of the label text, but with this update there are now two occurrences. ~So I updated the affected tests to use `getAllByLabelText` & grab the first occurrence.~ I updated the tests to use placeholders + `getByPlaceholderText` queries instead.

## Testing instructions

- Review release notes to check for breaking changes: https://github.com/carbon-design-system/carbon/releases/tag/v10.10.0
- Review each component in the storybook deploy & compare to our current release.
